src/components/InstallTabs/InstallTabs.scss

```diff
@@ -13,6 +13,7 @@
     border: 1px solid transparent;
     border-bottom: none;
     bottom: -1px;
+    color: var(--black4);
```

Comment:
For inactive tabs, can we put a little bit dark color maybe? RN both inactive and active seem to have same shade of text color

Reply:
@manishprivet great suggestion, I think that could be added to a new issue separately for better reference. We are going to start adding a few new accessibility issues hopefully very soon 👍

Reply:
I think this is fine until we run some automated accessibility checks.